packages/core-ui/src/modules/insights/containers/chart/ChartFormField.tsx

```
@@ -1,117 +1,141 @@
     }
 
     for (const logic of logics) {
-      const { logicFieldName, logicFieldValue, logicFieldVariable, logicFieldOperator } = logic;
+      const {
+        logicFieldName,
+        logicFieldValue,
+        logicFieldVariable,
+        logicFieldOperator,
+      } = logic;
 
       if (props.hasOwnProperty(logicFieldName) && props[logicFieldName]) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)